core/input/input.h

```diff
@@ -53,6 +53,7 @@ class Input : public Object {
 		MOUSE_MODE_CAPTURED,
 		MOUSE_MODE_CONFINED,
 		MOUSE_MODE_CONFINED_HIDDEN,
+		MOUSE_MODE_MAX,
 	};
```

Comment:
Since this can't be directly linked to `DisaplyServer` header, it might be worth adding `// Keep synced with "DisplayServer::MouseMode" enum.` comment.

Reply:
I followed your next suggestion.

Reply:
Well, I'll do both.